amp-story/templates/ampmars/index.html

```diff
@@ -0,0 +1,967 @@
+            height="1280"
+          ></amp-img>
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="vertical" class="mars-wrapper">
+          <amp-img
+            class="mars"
+            src="./img/03/mars.png"
+            width="960"
+            height="450"
+            layout="responsive"
+          ></amp-img>
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="vertical" class="suitcase-wrapper" >
+          <div>
+            <amp-img
+              class="suitcase"
+              src="./img/03/suitcase.png"
+              layout="responsive"
+              width="960"
+              height="663"
+              animate-in="fly-in-bottom"
+              animate-in-duration="0.85s"
+              animate-in-delay="0.45s"
+            ></amp-img>
+            <amp-img
+              class="books"
+              src="./img/03/books.png"
+              layout="responsive"
+              width="960"
+              height="663"
+              animate-in="fly-in-top"
+              animate-in-duration="0.75s"
+              animate-in-delay="0.8s"
+            ></amp-img>
+            <amp-img
+              class="sextant"
+              src="./img/03/sextant.png"
+              layout="responsive"
+              width="960"
+              height="663"
+              animate-in="fly-in-top"
+              animate-in-duration="1.25s"
+              animate-in-delay="1.2s"
+            ></amp-img>
+            <amp-img
+              class="helmet"
+              src="./img/03/helmet.png"
+              layout="responsive"
+              width="960"
+              height="680"
+              animate-in="fly-in-top"
+              animate-in-duration="0.75s"
+              animate-in-delay="1.1s"
+            ></amp-img>
+            <amp-img
+              class="suitcase-front"
+              src="./img/03/suitcase-front.png"
+              layout="responsive"
+              width="960"
+              height="663"
+              animate-in="fly-in-bottom"
+              animate-in-duration="0.85s"
+              animate-in-delay="0.45s"
+            ></amp-img>
+          </div>
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="vertical" class="copy-layer">
+          <h2
+            animate-in="fly-in-bottom"
+            animate-in-duration="0.5s"
+            animate-in-delay="1.05s"
+          >
+            what to bring
+          </h2>
+          <p
+            animate-in="fade-in"
+            animate-in-duration="0.6s"
+            animate-in-delay="1.55s"
+          >
+            Aside from warm clothes, there are a few essentials you’ll
+            want to bring from home: like a space suit, solar phone
+            charger, sextant, and a good book
+          </p>
+        </amp-story-grid-layer>
+      </amp-story-page>
+      <amp-story-page id="trip-highlights">
+        <amp-story-grid-layer template="fill">
+          <amp-video
+            controls
```

Comment:
I don't think we would want controls on the video, right?

Reply:
Hmm, I had an issue before where there video wouldn't autoplay unless the controls attribute was present.  It seems to not have an issue now.  Removed 'em.